sample-node-config/aws/README.md

```diff
@@ -0,0 +1,234 @@
+# Example Hydra Node Infrastructure
+
+This directory contains some [Terraform](https://www.hashicorp.com/products/terraform) and AWS based infrastructure code to setup a single [Hydra node](https://hydra.family/head-protocol/docs/getting-started/installation) connected to a [Cardano node](https://docs.cardano.org/getting-started/installing-the-cardano-node) running on `preview` testnet. It's not a complete turnkey solution and requires some tweaking and parameterisation to be completely usable but we thought it would be good starting point for new Hydra users.
+
+## Configuring Hydra Node
+The [configuration script](./scripts/configure-testnet.sh) assumes this image must be a reasonably recent Ubuntu/Debian Linux distribution, with [docker](https://docker.io) and [docker-compose](https://docs.docker.com/compose/) installed, and a configured user `ubuntu` with `sudo` access.
+
+The configuration process expects to find some files which are not provided by default and which are required for starting the Hydra node:
+* A Hydra signing key file `hydra-key.sk` which will be used in the Head to sign snapshots.
+  This can be generated using [hydra-tools](https://hydra.family/head-protocol/docs/getting-started/quickstart#hydra-keys),
+* A cardano signing key file  `cardano-key.sk` which is required to identify the parties on-chain and sign transactions.
+  This is a standard Cardano key so one can reuse an existing key or [generate a new one](https://hydra.family/head-protocol/docs/getting-started/quickstart#cardano-keys),
+* 0 or more hydra verification keys and cardano verification keys for the other Head parties,
+* The IP addresses and ports of _peer_ nodes,
+* Configuration files for [promtail](https://grafana.com/docs/loki/latest/clients/promtail/) and [prometheus](https://prometheus.io/) which are run as part of the stack,
+* Configuration files for the off-chain ledger.
```

Comment:
Could be cool to give the file here

Reply:
those files are already provided in the [docker-compose.yaml](https://github.com/input-output-hk/hydra-poc/blob/74c451e08d3eb041b94e113fd63a339899bb1ba5/sample-node-config/aws/docker/docker-compose.yaml#L54).
do you think we should mention them here?